src/registrar/models/dns/dns_soa.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from ..utility.time_stamped_model import TimeStampedModel
+from django.core.validators import MaxValueValidator, MinValueValidator
+
+
+class DnsSoa(TimeStampedModel):
+    mname = models.CharField(null=True, blank=True, default=None)
+
+    rname = models.CharField(blank=True, default="dns.cloudflare.com")
+
+    refresh = models.PositiveIntegerField(
```

Comment:
Yay for PositiveIntegerField! I didn't know that was available, but definitely helps with not having to write that constraint